inst/stan/epinowcast.stan

```diff
@@ -213,6 +269,18 @@ generated quantities {
           pp_inf_obs[i, k] += sum(pp_obs_tmp[snap, (sl[snap]+1):dmax]);
         }
       }
+      // cases with missing reference date (by estimated reference date)
+      for (i in 1:t) {
+        int snap = ts[i, k];
+        pp_inf_obs_miss[i, k] = sum(pp_obs_tmp_miss[snap, 1:dmax]);
+      }
+      // cases with missing reference date (by reporting date)
+      for (i in 2:t) {
+        int snap = ts[i, k];
+        for (l in max(dmax - i + 2, 1):min(dmax, t - i + 1)){
+          pp_inf_obs_miss_rep[(i - dmax) + l - 1, k] += pp_obs_tmp_miss[snap, l];
+        }
+      }
```

Comment:
Again, we currently cannot include the maximum delay cases and are thus slightly underpredicting the `obs_miss`.